judge/bridge/judgelist.py

```diff
@@ -99,12 +99,14 @@ def check_priority(self, priority):
     def judge(self, id, problem, language, source, priority):
         with self.lock:
             if id in self.submission_map or id in self.node_map:
-                logger.warning('Already judging? %d', id)
+                # Already judging, don't queue again. This can happen during batch rejudges, rejudges should be
```

Comment:
Removed this warning because despite filtering earlier (which is useful for showing users accurate submission counts), we can still race to this point.